be/src/formats/parquet/scalar_column_reader.cpp

```diff
@@ -18,10 +18,32 @@
 #include "gutil/casts.h"
 #include "io/shared_buffered_input_stream.h"
 #include "simd/simd.h"
+#include "statistics_helper.h"
 #include "utils.h"
 
 namespace starrocks::parquet {
 
+Status FixedValueColumnReader::row_group_zone_map_filter(const std::vector<const ColumnPredicate*>& predicates,
+                                                         SparseRange<uint64_t>* row_ranges,
+                                                         CompoundNodeType pred_relation, const uint64_t rg_first_row,
+                                                         const uint64_t rg_num_rows) const {
```

Comment:
For type `uint64_t`, no need to add const

Reply:
just make ide happy